pallets/parachain-staking/src/mock.rs

```diff
@@ -87,7 +87,7 @@ impl frame_system::Config for Test {
 	type MaxConsumers = frame_support::traits::ConstU32<16>;
 }
 parameter_types! {
-	pub const ExistentialDeposit: u128 = 1;
+	pub const ExistentialDeposit: u128 = 0;
```

Comment:
Why we use "insecure_zero_ed" and change this setting? Is non-zero setup conflicting test code?

Reply:
Since polkadot v0.9.42, ED will be required in addition to any amount to be reserved
see https://forum.polkadot.network/t/polkadot-release-analysis-v0-9-41-v0-9-42/2828

See also https://github.com/moonbeam-foundation/moonbeam/commit/93548eeba1e32361c401c58c87596b400707557a#diff-75d20fd33609e197547fb76a68dd4f5cd26e23e4e8f787295691e2869d672fd9

Reply:
My wonder is, why should we change our setup to insecure_zero_ed? Should not we change the test case to suit the non-zero ED instead?

Reply:
Anyway, my point is, "insecure_zero_ed" looks like an special branch for parachain who require a ED=0 in their runtime, like Moonbeam team. Without this feature, ED=0 is not allowed.

We do not implement ED=0 so it would be better not using this feature in test, right?

Reply:
> My wonder is, why should we change our setup to insecure_zero_ed? Should not we change the test case to suit the non-zero ED instead? Like giving extra initial balance

Probably because that would be a bit too much work (for now).

I changed that in `pallet-drop3` btw, but there're significantly more cases in parachain-staking.
My understanding is it only changes the behavior when you e.g. reverse your full balance - in the reality you would get the error already when doing so.

But again, I'm down to changing it to play it safe.
Can you please create an issue for that?

Reply:
I think I have changed all test cases already (by adding the initial balance!) That's why I wonder you enforce the "insecure_zero_ed". 

I will revert to normal pallet_balance feature and correct other potential errors?

Reply:
You will get panic without ED == 0 - you could revert the change and test it locally :)